frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -94,6 +94,13 @@ public ModelArchive registerModel(
             archive.getManifest().getModel().setHandler(handler);
         }
 
+        if (archive.getManifest().getModel().getHandler() == null
```

Comment:
This should be in else condition

Reply:
I added this to "else if". The reason for this "if" is that handler is given in the POST call, then it takes the highest precedence. Else, we should rely on the handler coming through the Manifest. If the MANIFEST doesn't provide one, then we should default to the "default handler".